Project one/Index.html

```diff
@@ -0,0 +1,62 @@
+<!Doctype html>
+<html>
+<link href="style.css" rel="stylesheet" type="text/css">
+
+<body>
+    <div class="main-nav">
+    <a id=grading-element-five>CONTACT</a> 
+        </div>
+    <a id="grading-element-two" href="about.html"> HOME </a>    
+    </div>
+
+    <br>
+    <br>
+
+    <button id="grading-element-one">Dont press me!</button>
+    <p id="con goes here"></p>
+
+    <button id="grading-element-three">Nothing to see here...</button>
+    <p id="grading-element-three-consequence" hidden> Told you there was nothing to see here...</p>
+
+    
+    <script>
+        document.getElementById("grading-element-one").addEventListener("click", youpressedme);
+        function youpressedme() {
+            var grading_element_one_consequence = document.createElement("p");
+            grading_element_one_consequence.setAttribute("id","grading-element-one-consequance")
+            grading_element_one_consequence.innerText = "Well Hello there";
+            document.getElementById("con goes here").appendChild(grading_element_one_consequence);
+        }
+        document.getElementById("grading-element-three").addEventListener("click", sneakyStuff);
+        function sneakyStuff(){
+            document.getElementById("grading-element-three-consequence").removeAttribute("hidden");
```

Comment:
i was looking for you to change `visibility: hidden;` to `visibility: visible` but this also works. Works as expected.